core/src/main/resources/error/error-classes.json

```diff
@@ -94,6 +94,11 @@
     ],
     "sqlState" : "22005"
   },
+  "COLUMNS_NUM_MISMATCH" : {
```

Comment:
`NUM_COLUMNS_MISMATCH`

Reply:
Not sure about this. For me, it means columns that contains numbers but columns number. If `NUM` comes first, it should `NUM_OF_COLUMNS_MISMATCH`, IMHO.

Reply:
Here is the pattern have used throughout my career: for (colNum = 0; colNum < numCols; colNum++).
I have no objection to NUM_OF_COLUMNS_MISMATCH or NUMBER_OF_COLUMNS_MISMATCH. The error-class name is not too long.

Reply:
ok. I will rename it.